dev/deps/spark-deps-hadoop-2.4

```diff
@@ -0,0 +1,185 @@
+JavaEWAH-0.3.2.jar
```

Comment:
Do we have to maintain these lists? Can't we use Maven to generate them from master, generate from PR, and compare?

Or is that an expensive operation?